### PR TITLE
release 0.4.0 - set uid field, be specified dependency for omniauth 1.9

### LINKED ISSUE #13: Auth Hash uid field is nil
I was expected that the auth hash `uid` field is set, but actually the value was returned nil.
It is bug.

actual returned auth hash :
```json
{
    "result": {
        "provider": "timetree",
        "uid": null,
        "info": {},
        "credentials": {
            "token": "FOO_BAR_TOKEN",
            "expires": false
        },
        "extra": {
            "raw_info": {
                "data": {
                    "id": "1234567890",
                    "type": "user",
                    "attributes": {
                        "name": "FOO BAR NAME",
                        "description": "FOO BAR DESCRIPTION",
                        "image_url": "https://attachments.timetreeapp.com/user/xxx.png"
                    }
                }
            }
        }
    }
}
```

### PR DESCRIPTION
**Pull request by koshilife**
- fix #13 set uid field.
- refs #12 be specified dependency for omniauth 1.9.

### CODE REVIEW

**Comment by koshilife** [comment]
I tested manually in remote server with https communication environment, and confirmed below.
  - should return uid valid value in auth hash.
  - should return credentials in auth hash. (regression scenarios)

@bonty Could your review this? If there is no problem, please merge.

File: lib/omniauth/strategies/timetree.rb [hunk 1]
```diff
@@ -24,6 +24,13 @@ def raw_info
         @raw_info = access_token.get(endpoint, options).parsed
       end
 
+      def extract_uid
+        return unless raw_info
+        return unless raw_info['data']
+
+        raw_info['data']['id']
```
Comment (bonty) [inline lib/omniauth/strategies/timetree.rb]: nits
I think using `dig` will be more simple.

```suggestion
        raw_info.dig('data', 'id')
```

**Review by bonty — APPROVED** [review]
I have commented just about coding style. But it's ok to merge without modifying. Thank you for fix!

**Comment by koshilife** [comment]
@bonty thanks for helpful review and suggestion!
I merged.

**Pull request merged**